matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/DefaultCryptoService.kt

```diff
@@ -775,6 +776,10 @@ internal class DefaultCryptoService @Inject constructor(
                     crossSigningService.onSecretUSKGossip(secretContent.secretValue)
                     return
             }
+            KEYBACKUP_SECRET_SSSS_NAME -> {
+                keysBackupService.onSecretKeyGossip(secretContent.secretValue)
+                return
```

Comment:
I think we don't need to "return" inside the when statement. It also seems ugly to have many return statements in a function and it may be difficult to debug. Although I understand that we need as many logs as possible I can suggest moving these checks to another function like checkReceivedSecretSendParameters().

Reply:
done